tests/fixtures/esp_decision_assistant/fallbacks_and_commondef_values.hocon

```diff
@@ -47,7 +52,9 @@ If you have all the information you need, there is no need to confirm the contex
                     # Gist says we are going to ask an LLM about how the response
                     # matches up to our description of acceptance criteria.
                     # Each component of the list needs to pass muster in order to pass.
-                    "gist": ["The text sample should generally reflect a negative sentiment about crossing the road."]
+                    "gist": ["""
+The text sample should generally reflect a negative sentiment about crossing the road in the immediate future.
+                             """"]
```

Comment:
Be more specific. Test was sometimes passing "wait" as not being negative, so add more about immediacy.
Seems to be doing better. 40/40 passing now.